src/app/browse-by/browse-by-switcher/themed-browse-by-switcher.component.ts

```diff
@@ -17,7 +17,7 @@ export class ThemedBrowseBySwitcherComponent extends ThemedComponent<BrowseBySwi
   }
 
   protected importThemedComponent(themeName: string): Promise<any> {
-    return import(`../../../themes/${themeName}/app/+browse-by/+browse-by-switcher/browse-by-switcher.component`);
+    return import(`../../../themes/${themeName}/app/browse-by/+browse-by-switcher/browse-by-switcher.component`);
```

Comment:
@artlowel : Still seeing a "+browse-by-switcher" in this line.  That has been renamed but this import needs a quick fix.

Reply:
Good catch! 
Fixed